File: ServerSide/SAMS.BusinessLogic/Models/Faculty/FacultyGridModel.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace SAMS.BusinessLogic.Models.Faculty
{
    public class FacultyGridModel
    {
        public int Id { get; set; }
        public string Code { get; set; }
        public string Title { get; set; }
        public int StudyProgrammeCount { get; set; }
    }
}

File: ClientSide/src/app/pages/faculties/models/facultyGridModel.ts
export class FacultyGridModel {

    public id: number;
    public code: string;
    public title: string;
    public studyProgrammeCount: number;

    constructor(
        id: number,
        code: string,
        title: string,
        studyProgrammeCount: number
    ) {
        this.id = id;
        this.code = code;
        this.title = title;
        this.studyProgrammeCount = studyProgrammeCount;
    }

}
